pandas/core/indexes/base.py

```diff
@@ -4021,8 +4021,56 @@ def unique(self, level=None):
     def drop_duplicates(self, keep='first'):
         return super(Index, self).drop_duplicates(keep=keep)
 
-    @Appender(base._shared_docs['duplicated'] % _index_doc_kwargs)
     def duplicated(self, keep='first'):
+        """
+        Indicate duplicate index values.
+
+        Duplicated values are indicated as ``True`` values in the resulting
```

Comment:
can you coordinate text with #20114, seems some slight differences

Reply:
Looks like they are in the zone together already :-) 
(they are sitting close to me: we removed the extended summary in the other PR as Tom asked, or are there other differences?)

Reply:
great, seemed likely.